falcon/api.py

```diff
@@ -469,10 +469,6 @@ def _get_responder(self, req):
             method was not found in the method_map for the responder,
             the responder callable element of the returned tuple will be
             `falcon.responder.bad_request`.
```

Comment:
Seems like if we are going to do this for 404, we should also do it for 400. What do you think?